mlir/unittests/IR/AttributeTest.cpp

```diff
@@ -474,7 +475,9 @@ TEST(CopyCountAttr, CopyCount) {
   test::CopyCount::counter = 0;
   test::TestCopyCountAttr::get(&context, std::move(copyCount));
 #ifndef NDEBUG
-  // One verification enabled only in assert-mode requires a copy.
+  // One verification enabled only in assert-mode requires two copies: one for
+  // calling 'verifyInvariants' and one for calling 'verify' inside
+  // 'verifyInvariants'.
```

Comment:
I believe the comment should also be updated, since there is only one copy now. It should be sufficient to keep the original comment (and update the same comment in the `CopyCountGetChecked` test).